cloudflare/resource_cloudflare_access_policy.go

```diff
@@ -97,6 +97,13 @@ var policyOptionElement = &schema.Resource{
 			Type:     schema.TypeBool,
 			Optional: true,
 		},
+		"group": {
```

Comment:
We'll need to add an integration test for this new property. If you have a look at `resource_cloudflare_access_policy_test.go` there already some examples in there.

Reply:
Done